src/DebugMessages.h

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (C) 1996-2020 The Squid Software Foundation and contributors
```

Comment:
If you are going to update this PR for other reasons, let's update the copyright to match the current year.

```suggestion
 * Copyright (C) 1996-2021 The Squid Software Foundation and contributors
```

BTW, if you have not formatted PR-changed sources, please do so.

Reply:
Done and autoformatted.